spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -143,6 +143,61 @@
         expect(whodunnit_actor).to be_a(User)
         expect(whodunnit_actor.id).to eq(user.id)
       end
+
```

Comment:
These tests are about making sure the log gets the correct managing and owning orgs correctly auto-assigned (or not auto-assigned where appropriate).